tests/builder/help.rs

```diff
@@ -964,6 +999,71 @@ OPTIONS:
     ));
 }
 
+#[test]
+#[cfg(all(feature = "wrap_help"))]
+fn possible_value_wrapped_help() {
+    #[cfg(feature = "unstable-v4")]
+    static WRAPPED_HELP: &str = "test 
+
+USAGE:
+    test [OPTIONS]
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+        --possible-values <possible_values>
+            Possible values:
+                - name: Long enough help message to clearly
+                          warrant wrapping
+                - second
+
+        --possible-values-with-new-line <possible_values_with_new_line>
+            Possible values:
+                - long enough name to trigger new line:
+                    Long enough help message to clearly warrant
+                    wrapping
+                - second
```

Comment:
Should we automatically use next-line if we need to wrap?

Reply:
> Should we automatically use next-line if we need to wrap?

I implemented it similarly to the help messages of arguments, only that the decision of nexline/sameline is local to an argument, and instead of indenting on tab it puts `: ` there. 

I actually noticed a bug in this test case, I wanted the help messages to align, so the correct output would be (so that a help message of second would align):
```suggestion
                - name:   Long enough help message to clearly
                          warrant wrapping
                - second

        --possible-values-with-new-line <possible_values_with_new_line>
            Possible values:
                - long enough name to trigger new line:
                    Long enough help message to clearly warrant
                    wrapping
                - second
```

Reply:
> I implemented it similarly to the help messages of arguments, only that the decision of nexline/sameline is local to an argument, and instead of indenting on tab it puts : there.

This is copying the short help logic.  What the right decision is though, is unclear.  Let's just see how this goes and what feedback we get.

Reply:
1. use the same alignment for all arguments, i.e. calculate it beforehand
2. use the same mode nextline/sameline for all arguments
3. change style to: `{TAB_WIDTH}{name}{TAB_WIDTH+longest-display_width(name)}{help}`
4. Always switch to nextline for wrapping help messages instead of triggering it by long names

Reply:
Looking at your options and the output, I'm leaning towards (initially):
- Don't be consistent across args
- Do be consistent within an arg
- Treat long args and long abouts the same (both using next-line)